### ISSUE TITLE
requests package needed but not added as dependency

### ISSUE THREAD

**Issue by sasa-buklijas** [mzdrale/pyubee#27]
```
$ pyubee -h
Traceback (most recent call last):
  File "/.../9_no_git_testing/pyubee/venv-3_12_0/bin/pyubee", line 5, in <module>
    from pyubee.__main__ import main
  File "/.../9_no_git_testing/pyubee/venv-3_12_0/lib/python3.12/site-packages/pyubee/__init__.py", line 8, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

`requests` package needed but not added as dependency.   
Could do it, but do not know what version to set.

**Comment by mzdrale** [comment]
Thanks @sasa-buklijas for creating issue.

If you have time, please add it and create PR. I would use the latest requests==2.31.0.
Thanks.

**Comment by sasa-buklijas** [comment]
do you want as requests==2.31.0 or requests>=2.31.0 ?

**Comment by mzdrale** [comment]
I would pin it to the specific version, so `requests==2.31.0`. Thanks for contributing. :)

**Comment by sasa-buklijas** [comment]
done https://github.com/mzdrale/pyubee/pull/30

**Comment by mzdrale** [comment]
Thank you. Changes merged and new [0.13 version](https://pypi.org/project/pyubee/0.13/) released.
